reclutch/src/lib.rs

```diff
@@ -5,8 +5,8 @@
 //!
 //! # `Widgets`
 //!
-//! A widget only defines 3 methods; [`bounds`](trait.Widget.html#tymethod.bounds),
-//! [`update`](trait.Widget.html#tymethod.update), and [`draw`](trait.Widget.html#tymethod.draw).
+//! A widget only defines 3 methods; [`bounds`](widget::Widget::bounds),
+//! [`update`](widget::Widget::update), and [`draw`](widget::Widget::draw).
```

Comment:
Do these actually resolve in stable Rust? I tried this syntax of intradoc linking previously and it didn't work.

Reply:
ok, I tried it and it failed, but the other way seems to fail sometimes, too.

Reply:
I fixed it.